commons/dev/configmaps/flyway-traces-schema.yaml

```
@@ -27,3 +27,7 @@ data:
       producer_external_id UUID,
       created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
     );
+
+  V1.0.1__Traces_Schema.sql: |-
```

```suggestion
  V1.1.0__Token_UUID.sql: |-
```
given the nature of the change, I'd say it's a minor version bump. Also changed the description part for clarity

---

absolutely correct! appreciated :)